protobuf_definitions/aquatroll.proto

```diff
@@ -296,7 +327,7 @@ message AquaTrollParameterBlock {
   float measured_value = 1;
   AquaTrollParameter parameter_id = 2;
   AquaTrollUnit units_id = 3;
-  AquaTrollQuality data_quality_id = 4;
+  repeated AquaTrollQuality data_quality_ids = 7;
```

Comment:
Nice to do it like this to be compatible. Additionally, you should also add the old ids (and possibly names) to the reserved fields.

https://protobuf.dev/programming-guides/proto3/#fieldreserved